subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorFragments.kt

```diff
@@ -65,12 +62,13 @@ fun fragmentsFor(accessor: Accessor): Fragments = when (accessor) {
 private
 fun fragmentsForConfiguration(accessor: Accessor.ForConfiguration): Fragments = accessor.run {
 
-    val propertyName = name.original
+    val propertyName = config.target.original
```

Comment:
⭕️ In order to minimize the number of changes, a `name` variable could be declared here:

```
val name = config.target
```

Then most of `name => config.target` changes below become unnecessary.